interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -1186,71 +1186,77 @@ function ViewModel() {
+            if (!ko.utils.arrayFirst(self.allMessages(), function(item) { return item.index == 'OrphanedMsg' })) {
                 self.allMessages.push({
                     index: 'OrphanedMsg',
                     type: glitterTranslate.status['INFO'],
                     text: glitterTranslate.orphanedJobsMsg + ' <a href="#" onclick="showOrphans()"><span class="glyphicon glyphicon-wrench"></span></a>',
                     css: 'info',
-                    clear: function() { self.clearMessages('OrphanedMsg')}
+                    clear: function() { self.clearMessages('OrphanedMsg') }
                 });
             }
         } else {
             // Remove any message, if it was there
             self.allMessages.remove(function(item) {
-               return item.index == 'OrphanedMsg';
+                return item.index == 'OrphanedMsg';
             })
         }
     })
 
     // Message about localStorage not being enabled every 20 days
-    if(!hasLocalStorage && localStorageGetItem('LocalStorageMsg')*1+(1000*3600*24*20) < Date.now()) {
+    if (!hasLocalStorage && localStorageGetItem('LocalStorageMsg') * 1 + (1000 * 3600 * 24 * 20) < Date.now()) {
         self.allMessages.push({
             index: 'LocalStorageMsg',
             type: glitterTranslate.status['WARNING'].replace(':', ''),
             text: glitterTranslate.noLocalStorage,
             css: 'warning',
-            clear: function() { self.clearMessages('LocalStorageMsg')}
+            clear: function() { self.clearMessages('LocalStorageMsg') }
         });
     }
 
-    document.onkeydown = function(e) {
-        if(self.keyboardShortcuts()) {
-            // Ignore if the user used a combination
-            if(e.altKey || e.metaKey || e.ctrlKey) return;
-
-            // Do not act if the user is typing something
-            if($("input:focus, textarea:focus").length === 0) {
-                if (e.code === 'KeyP') {
-                    self.pauseToggle();
-                }
-                if (e.code === 'KeyA') {
-                    $('#modal-add-nzb').modal('show');
-                }
-                if (e.code === 'KeyC') {
-                    window.location.href = './config/';
-                }
-                if (e.code === 'KeyS') {
-                    self.loadStatusInfo(true, true)
-                    $('#modal-options').modal('show');
-                }
-            }
-
-        }
+    if (self.keyboardShortcuts()) {
+        $(document).bind('keydown', 'shift+p', function(e) {
+            self.pauseToggle();
+        });
+        $(document).bind('keydown', 'shift+a', function(e) {
+            $('#modal-add-nzb').modal('show');
+        });
+        $(document).bind('keydown', 'shift+c', function(e) {
+            window.location.href = './config/';
+        });
+        $(document).bind('keydown', 'shift+s', function(e) {
```

Comment:
Why include the `shift`? The current version works without shift, so why change?

Reply:
Current version for me does it on key or shift+key combo. So when adding the arrow keys for the pagination, it would break accessibility if we were using the standard arrow keys without using the shift modifier. So using shift for all keeps things consistent.

Reply:
I understand for the pagination that we should use the shift, but the current release uses non-shift for the P/A/S/C commands, so let's keep it that way?

Reply:
The current way works with shift and without

Reply:
Only without shift for the existing shortcuts, with shift for the new pagination ones.